packages/apps/tasks/src/TaskList.tsx

```diff
@@ -0,0 +1,128 @@
+import { ShellLayout } from "@dxos/react-client";
+import { useIdentity } from "@dxos/react-client/halo";
+import { useQuery, useSpaces } from "@dxos/react-client/echo";
+import React, { useState } from "react";
+
+import { Task } from "./proto";
+import { useShell } from "@dxos/react-shell";
+
+export const TaskList = () => {
+  useIdentity({ login: true });
+  const [space] = useSpaces(); // What should the pattern be for find-or-create a space?
```

Comment:
This is supposed to get better with progressive multiplayer and default spaces. But in the meantime, I think I'd like to "find or create" a space with a specific name (`dxos-tasks-app`?) that could be used just for this app's data.

Reply:
At one point in time @zhenyasav had written a hook for that, not sure where it ended up